examples/sign_info.py

```diff
@@ -77,9 +81,8 @@
                 "actions": [
                     {
                         "action": "c2pa.created",
-                        "parameters": {
-                            # could hold additional information about this step
-                        }
+                        "digitalSourceType": "http://cv.iptc.org/newscodes/digitalsourcetype/digitalCreation",
+                        "softwareAgent": "Python C2PA SDK example",
```

Comment:
see note above on softwareAgent (FYI this was deprecated in v1 claims long ago, so we need to move forward.